### ISSUE TITLE
"XML Parsing Error" in FireFox

### ISSUE THREAD

**Issue by norin89** [FWeinb/grunt-svgstore#77]
I created pretty huge SVG icon set for one of my project. The issue is that some of the icons use `xlink:href` itself. I mean there is a simple file `a.svg` like this:

``` svg
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 64 64">
    <rect x="4" y="4" width="56" height="56" />
</svg>
```

and I use it in another file `b.svg` like this:

``` svg
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 64 64">
    <use xlink:href="#icon-a" />
</svg>
```

The problem is that FireFox throws "XML Parsing Error: prefix not bound to a namespace" because of lack of `xmlns:xlink` attribute. As I can see there is only an option to define `xlink` parameter but not any of other namespaces like mentioned `xmlns:xlink`, `xmlns:xsl' or any others.

I did some work around with other grunt task [grunt-string-replace](https://github.com/erickrdch/grunt-string-replace) to add missing namespace but that would be nice to be able to declare that inside the task.

**Comment by lambert-velir** [comment]
Hi-  Did you ever find a solution to this?  I'm using gulp-svgstore, but running into the same problem.

**Comment by norin89** [comment]
No. I still use my workaround using [grunt-string-replace](https://github.com/erickrdch/grunt-string-replace) and adding required attribute.

**Comment by alvarobyrne** [comment]
Hi. Sure enough I have the same problem. @norin89 : could you please be a bit more explicit into what's the precise replacement one should do when using grunt-string-replace. Thanks in advance for your help.

**Comment by norin89** [comment]
First I run `grunt-svgstore` and build SVG sprite to temporary location. Then I use [grunt-text-replace](https://github.com/yoniholmes/grunt-text-replace) (I switched from `grunt-string-replace`), take SVG from temp location replace `xmlns` attribute to add `xmlns:xlink` and save task in build directory.

Here's a snippet of my replacement task:
```javascript
		'replace': {
			svg: {
				src: ['./.temp/sprite.svg'],
				dest: './build/',
				replacements: [{
					from: 'xmlns="'+'<%= svgIcons.xmlns %>'+'"',
					to: 'xmlns="'+'<%= svgIcons.xmlns %>'+'" xmlns:xlink="http://www.w3.org/1999/xlink"'
				}]
			}
		}
```

I store `xmlns` attribute as variable cause I add it to SVG with `option.svg.xmlns` parameter of `grunt-svgstore` task but you can also omit this parameter and replace `<svg` with `<svg xmlns="..." xmlns:xlink="..."`.

**Comment by alvarobyrne** [comment]
@norin89 thanks for your help
